packages/axios-extension/src/auth/uaa.ts

```diff
@@ -86,6 +86,24 @@ export class Uaa {
         return this.serviceInfo.systemid;
     }
 
+    /**
+     * Getter for resource owner.
+     *
+     * @returns username
+     */
```

Comment:
Do we need doc comments for private getters? The comments are longer than the code